test/msw-api/setup-worker/scenarios/shared-worker/shared-worker.test.ts

```diff
@@ -0,0 +1,29 @@
+import * as path from 'path'
+import { pageWith } from 'page-with'
+import { waitFor } from '../../../../support/waitFor'
+
+function createRuntime() {
+  return pageWith({
+    example: path.resolve(__dirname, 'shared-worker.mocks.ts'),
+    contentBase: path.resolve(__dirname),
+  })
+}
+
+test('supports shared workers', async () => {
```

Comment:
Do I understand it correctly that the intention is not to intercept requests from shared workers in any way but rather not obstruct them if they do exist alongside MSW?

If it's the latter, we should reflect that in the test name.

Reply:
Intercepting the worker code itself also works with MSW after that. But that's working out of the box if this is fixed.

Reply:
Thanks for clarifying!